CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -1728,13 +1726,11 @@ extern    "C"
 
   /**
    * @brief Get the required buffer size for S8 average pooling function
-   * @param[in]       dim_dst_width         output tensor dimension
-   * @param[in]       ch_src                number of input tensor channels
+   * @param[in]       input_ch              number of input tensor channels
    * @return          The function returns  required buffer size
    *
    */
-    int32_t arm_avgpool_s8_get_buffer_size(const int dim_dst_width,
-                                           const int ch_src);
+    int32_t arm_avgpool_s8_get_buffer_size(const int input_ch);
```

Comment:
Please put the old API back(Though dim_dst_width is not used now) to avoid frequent API updates if it is not absolutely necessary.

Reply:
Rest look good!

Reply:
Ok, will do